modules/document_repository/ajax/GetFile.php

```diff
@@ -35,27 +34,33 @@
 // Checks that config settings are set
 $config =& NDB_Config::singleton();
 
-$File = $_GET['File'];
+$file = $_GET['File'];
```

Comment:
`basename` won't work here because for whatever reason this module is coded to retrieve files by "`$user/$filename`" instead of `$filename`.

I do ultimately think that the `basename` approach is better but it would involve a more in-depth refactoring of this module.